packages/commonwealth/client/scripts/state/api/threads/helpers/counts.ts

```diff
@@ -0,0 +1,14 @@
+import { ThreadStage } from 'models/types';
+import { EXCEPTION_CASE_threadCountersStore } from '../../../ui/thread';
+
+export const updateThreadCountsByStageChange = (currentStage, updatedStage) => {
```

Comment:
Do we know what the purpose of this stage-based updating is?

Reply:
It existed before the migration and i don't know the exact purpose but from the code what I understood was whenever a thread goes from any stage to `voting` we update the total voting threads, similarly when its removed from `voting`, we decrement the count.